doc/environment_variables.md

```diff
@@ -49,6 +49,7 @@ some configuration as environment variables that can be set.
   - connectivity_state - traces connectivity state changes to channels
   - channel_stack_builder - traces information about channel stacks being built
   - executor - traces grpc's internal thread pool ('the executor')
+  - fd_trace - traces fd creation, shutdown and closure.
```

Comment:
Make a note that this only works with epollex

Reply:
the traces in ev_posix.cc should capture fd_create/fd_close/fd_shutdown for all polling engines. The traces in epollex are only for epoll fd.

Reply:
ok i guess, I can just put this in the doc :). Will do.